lib/index.rb

```diff
@@ -19,7 +19,9 @@ def read_json(path)
   repo: ENV["GITHUB_REPOSITORY_NAME"] || @event_json.dig("repository", "name"),
 }
 
-@haml = "haml-lint " + ENV["INPUT_FILE_PATHS"]
+@input_file_paths = ENV["INPUT_FILE_PATHS"].empty? ? "app/views/" : ENV["INPUT_FILE_PATHS"]
```

Comment:
I guess we technically don't need to add this but I would rather default to `app/views` rather than `app/views/**/*.haml.html`

Reply:
Fair assumption! better to be safe then sorry